runner/lib/helpers/buffer-line-transform.js

```diff
@@ -0,0 +1,95 @@
+/* global Buffer */
+/* eslint-disable no-underscore-dangle,no-nested-ternary,no-plusplus */
+
+import { Transform } from 'stream';
```

Comment:
Ah, please discount my directions regarding the buffer type since you’re building on Node.js streams.

I have never been able to confidently implement much less review Node.js stream transforms so I’ve been building Uint8Array async iterator transforms instead.

Reply:
Yup this is definitely a node stream thing, and needs to stay that way to avoid converting Buffer into something else, which was the main performance issue.